sample/dockerwait.js

```diff
@@ -13,11 +13,10 @@ var bootCheckInterval = Number(process.env.SOCKETCLUSTER_BOOT_CHECK_INTERVAL) ||
 var bootTimeout = Number(process.env.SOCKETCLUSTER_CONTROLLER_BOOT_TIMEOUT) || 10000;
 var bootStartTime = Date.now();
 
-var errorMessage = `Failed to locate the master controller file at path ${masterControllerPath} ` +
-`before SOCKETCLUSTER_CONTROLLER_BOOT_TIMEOUT`;
+var errorMessage = `Failed to locate the master controller file at path ${masterControllerPath} before SOCKETCLUSTER_CONTROLLER_BOOT_TIMEOUT`;
```

Comment:
There are plenty of >80chars lines all over the codebase, so it doesn't make much sense to split error messages.